src/content/guides/farmlands-pos-integration.mdx

```diff
@@ -722,6 +722,10 @@ Note over POS: ✅ Display Release confirmation
     ```
 
 6. The POS [releases any remaining authorised funds](https://docs.centrapay.com/api/payment-requests#release-funds-held-for-a-pre-auth-payment-request-experimental) against the [Payment Request](https://docs.centrapay.com/api/payment-requests#payment-request) back to the Cardholder.
+Funds requiring release can either be tracked internally or you can [list payment activities](https://docs.centrapay.com/api/payment-requests#list-payment-activities-for-a-payment-request) and sum the confirmation [payment activities](https://docs.centrapay.com/api/payment-requests#payment-activity).
```

Comment:
IMO this line is self explanatory and does not need to be documented.

Reply:
This was requested by Brijesh to make it easier for an integrator to know how to know the remaining left on a pre auth. He wanted a whole section on it but I think this is enough. I don't think that it would be immediatly obvious how to get the remaining amount on a pre auth. Perhaps the part about tracking internally could be removed.

Reply:
Yes, I would remove the part about internal tracking.